src/terminal/pty/ConPty.cpp

```diff
@@ -84,6 +84,8 @@ ConPty::ConPty(PageSize const& _windowSize) :
 
     if (hr != S_OK)
         throw runtime_error{ GetLastErrorAsString() };
+
+    buffer_.resize(10240);
```

Comment:
The buffer_ here had size of 0, which prevents any read operation from happening.

Reply:
Ah yes! I just recently refactored that part on Pty in general, adding this `buffer_`, but didn't actually test on windows. Thanks!